### PR TITLE
Additional Documentation

### PR DESCRIPTION
**Pull request by Pickysaurus**
Created a new "Adding Games" page to user docs.
Improved the "Getting Started" page by adding buttons to the Windows/Linux download options and mentioning the Linux packages. 
Corrects several small mistakes in existing docs.

### CODE REVIEW

File: docs/users/SystemRequirements.md [hunk 1]
```diff
@@ -1,23 +1,52 @@
...
 The App requires the following programs to be available at runtime:
 
 - `xdg-open` to open URLs in your browser.
 - `update-desktop-database` to update the MIME cache after creating a desktop entry file for the NXM link handler.
 
 If you plan to install using the AppImage, make sure you have the following dependencies installed:
 
+- [FUSE 2](https://github.com/AppImage/AppImageKit/wiki/FUSE) is required to run any AppImage.
 - `glibc` 2.2.5 or newer
 - `glibc++` 3.4 or newer
```
Comment (erri120) [inline docs/users/SystemRequirements.md:43]: `glibc` and `glibc++` are always required. FUSE 2 is required for the AppImage.

File: docs/users/images/OAuthSignIn.png
Comment (erri120) [inline docs/users/images/OAuthSignIn.png:1]: This should be cropped to the content.

File: docs/users/GettingStarted.md [hunk 2]
```diff
@@ -1,26 +1,33 @@
...
-#### AppImage
+To get the most out of the app, we highly recommend signing in with your Nexus Mods account. If you don't have one yet, you can make one for free [here](https://users.nexusmods.com/register). 
 
-You can download the AppImage from the [latest release] on GitHub. Make sure you have [FUSE 2](https://github.com/AppImage/AppImageKit/wiki/FUSE) installed before running it.
+When you first open the app, you will have the option to sign in. If you skipped this step you can always get started by clicking the "Log In" button in the top-right of the application. 
 
-#### System Package Manager
+![The top bar navigation of the app while logged out.](./images/0.5.2/Topbar.png)
 
-If you want to create a package for your platform, see [Contributing](../developers/Contributing.md#for-package-maintainers) for more details.
```
Comment (erri120) [inline docs/users/GettingStarted.md:21]: This should be with the repology table.

File: docs/users/GettingStarted.md [hunk 3]
```diff
@@ -1,26 +1,33 @@
...
+Before installing the app, please ensure that your PC meets the system requirements for both [this app](./SystemRequirements.md) and the game you are modding.
 
 ## Installation
 
-### Windows
+Download the latest version of the app and run the installer on your system.
 
-Download and run the [Windows Installer] from the [latest release] on GitHub.
+[:fontawesome-brands-windows: Windows Installer](https://github.com/Nexus-Mods/NexusMods.App/releases/latest/download/NexusMods.App.x64.exe){ .md-button .md-button--primary }
+[:fontawesome-brands-linux: Linux AppImage](https://github.com/Nexus-Mods/NexusMods.App/releases/latest){ .md-button }
```
Comment (erri120) [inline docs/users/GettingStarted.md:12]: Should use forward defined links. The AppImage link can point to `https://github.com/Nexus-Mods/NexusMods.App/releases/latest/download/NexusMods.App.x86_64.AppImage`.

This should also mention that other version are available as well and link to the release itself at `https://github.com/Nexus-Mods/NexusMods.App/releases/latest`, as well as the repology table for system packages.

**Review by erri120 — DISMISSED** [review]

**Comment by erri120** [comment]
Images should also be WebP, binary blobs can make git checkout very slow over time.

File: docs/users/images/OAuthSignIn.png
Comment (Pickysaurus) [reply docs/users/images/OAuthSignIn.png:1]: No, it should not.

File: docs/users/GettingStarted.md (hunk 3 above)
Comment (Pickysaurus) [reply docs/users/GettingStarted.md]: I think you're getting bogged down a bit again. This is intended to provide the easiest possible way for users to get started. I've updated the AppImage link and added a link to "Other releases" below the CTA buttons but anything further is unnecessary.

File: docs/users/SystemRequirements.md (hunk 1 above)
Comment (Pickysaurus) [reply docs/users/SystemRequirements.md:43]: This is your text, I just copy/pasted it.

Comment (erri120) [reply docs/users/SystemRequirements.md:43]: It's the text above that's an issue

> If you plan to install using the AppImage, make sure you have the following dependencies installed:

Can probably just be something like this:

```markdown
The App requires the following dependencies:

- `glibc` 2.2.5 or newer
- `glibc++` 3.4 or newer
- [FUSE 2](https://github.com/AppImage/AppImageKit/wiki/FUSE) if you plan on using the AppImage.
```

File: docs/users/images/OAuthSignIn.png
Comment (erri120) [reply docs/users/images/OAuthSignIn.png:1]: Why not?

**Comment by erri120** [comment]
> > Images should also be WebP, binary blobs can make git checkout very slow over time. Make sure to use a rebase for this, so that the original PNGs aren't in the history.
> 
> There is not a good way to create WebPs on Windows. I can compress them as JPGs or we can split the docs off into a submodule?

I'll turn them into WebPs. Most editing software should be able to do this tho (Paint.NET, GIMP, Photoshop).

**Review by halgari — APPROVED** [review]

**Pull request merged**